### PR TITLE
translate `Imports match directories and files`

### PR DESCRIPTION
**Pull request by serihiro**
# What will this PR change?
- add Japanese translation for the content `Imports match directories and files` in `Imports and modules` section.
- English version text was added in https://github.com/dlang-tour/english/pull/181

### CODE REVIEW

File: basics/imports-and-modules.md [hunk 1]
```diff
@@ -19,6 +19,16 @@ Dでシンプルなhello worldプログラムを書くには`import`が必要で
 `import`文はソースファイルの一番上に出てくる必要はありません。
 関数やその他のスコープ内で局所的に使用することもできます。
 
+### インポートはディレクトリとファイルと一致します
+
+Dのモジュールシステムは、他のシステムとは対照的に、完全にファイルに基づいています。
+例えば、`my.cat`は常に`my/`フォルダの中にある`cat.d`というファイルを常に参照します。
```
Comment (simdnyan) [inline basics/imports-and-modules.md]: `常に` が冗長です。

File: basics/imports-and-modules.md [hunk 2]
```diff
@@ -19,6 +19,16 @@ Dでシンプルなhello worldプログラムを書くには`import`が必要で
...
+`my`フォルダはカレントディレクトリか明示的に指定されたディレクトリインポート(`-I`)のどれか１つに存在する必要があります。
+最後に、簡単に大きなモジュールを複数の小さなファイルに分割するために、`cat.d`の代わりに、`cat/`フォルダを同様に使うことができます。
+その場合Dコンパイラは`my/cat.d`の代わりに`my/cat/package.d`をロードしようとします。
+
+`package.d`ファイルの慣習（厳密なルールではありませんが）として、同じフォルダにある他の全てのモジュールをimportします。
```
Comment (simdnyan) [inline basics/imports-and-modules.md]: ここでは、`not a hard rule` と対になるので `慣習` ではなく `規約` かと思います。

**Review by simdnyan — COMMENTED** [review]
Thank you for your contributes! I added some comments. Please have a look.

**Comment by serihiro** [comment]
@simdnyan 
Thank you for your review!
I fixed and pushed. Could you check this again ?

The build of this commit is failed, but it seems not to relate to my commit(failed by wget error).

```
The command "wget https://raw.githubusercontent.com/dlang/tools/master/rdmd.d" exited with 4.
0.01s$ ${DC} rdmd.d
Error: module rdmd is in file 'rdmd.d' which cannot be read
import path[0] = /home/travis/dlang/ldc-1.5.0/bin/../import/ldc
import path[1] = /home/travis/dlang/ldc-1.5.0/bin/../import
The command "${DC} rdmd.d" exited with 1.
0.00s$ export PATH=$PATH:$PWD
```

**Pull request merged**